src/services/jsDoc.ts

```diff
@@ -68,6 +68,33 @@ namespace ts.JsDoc {
         return documentationComment;
     }
 
+    export function getJsDocTagsFromDeclarations(declarations: Declaration[]) {
+        // Only collect doc comments from duplicate declarations once.
+        const tags: JSDocTagInfo[] = [];
+        forEachUnique(declarations, declaration => {
+            const jsDocs = getJSDocs(declaration);
+            if (!jsDocs) {
+                return;
+            }
+            const jsDocTags: JSDocTag[] = [];
+            jsDocs.forEach(doc => {
+                const tagsForDoc = (doc as JSDoc).tags;

```

Comment:
I guess not. I got confused by the 2.0 language service giving me an implicit any warning...